src/server/protocol.ts

```diff
@@ -92,6 +92,7 @@ namespace ts.server.protocol {
         SynchronizeProjectList = "synchronizeProjectList",
         /* @internal */
         ApplyChangedToOpenFiles = "applyChangedToOpenFiles",
+        ApplyChangesToOpenFiles = "applyChangesToOpenFiles",

```

Comment:
Any suggestion on api name?

Reply:
How about `applyChanges` since the command takes opened and closed files too?

Reply:
How closely is it related to `change`?  Maybe `changeMultiple`?

Reply:
`applyChanges` also works for me - I just find it a little vague.  It's convenient for people who already know the name of the internal API but seems like it might be less clear to new consumers.

Reply:
I see that this is generalizing `open` and not `change`.  What about `updateOpen`?  I think `updateOpenFiles` would be clearer, but it looks like "file(s)" is implicit in our other command names.  `updateOpen` works for both updating the set of open files and the contents of open files.